app/helpers/view_helper.rb

```diff
@@ -14,4 +16,18 @@ def bat_contact_email_address
   def bat_contact_mail_to(name = nil, subject: nil, link_class: "govuk-link")
     mail_to bat_contact_email_address, name || bat_contact_email_address, subject: subject, class: link_class
   end
+
+  def subject_is_selected?(id:)
```

Comment:
These are specific to one page so I don't think they should go in the general `ViewHelper`. They should be in a specific `ResultFilters::SubjectHelper` or a view object used by that page.